docs/resources/sfs_turbo_obs_target.md

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "Scalable File Service (SFS)"
+---
+
+# huaweicloud_sfs_turbo_obs_target
+
+Manages an OBS target resource under the SFS Turbo within HuaweiCloud.
+
+-> The resource supported SFS Turbo file system types are **20MB/s/TiB**, **40MB/s/TiB**, **125MB/s/TiB**,
+  **250MB/s/TiB**, **500MB/s/TiB**, **1,000MB/s/TiB**, **HPC**.
+
+## Example Usage
+
+```hcl
+variable "share_id" {}
+variable "file_path" {}
+variable "bucket" {}
+variable "endpoint" {}
+
+resource "huaweicloud_sfs_turbo_obs_target" "test" {
+  share_id         = var.share_id
+  file_system_path = var.file_path
+
+  obs {
+    bucket   = var.bucket
+    endpoint = var.endpoint
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this creates a new resource.
+
+* `share_id` - (Required, String, ForceNew) Specifies the ID of the SFS Turbo file system to which the OBS target
+  belongs. Changing this creates a new resource.
+
+* `file_system_path` - (Required, String, ForceNew) Specifies the linkage directory name of the OBS target.
+  Changing this creates a new resource.
+
+  -> The directory name must be unique and it can not be **.** or **..**. The directory name can not contain
+    slashes **/** and multi level directory is not supported.
```

Comment:
```suggestion
  -> The directory name must be unique and it can not be `.` or `..` character. The directory name can not contain
    slashes (/) and multi level directory is not supported.
```